lib/phoenix_swagger/schema.ex

```diff
@@ -37,6 +37,7 @@ defmodule PhoenixSwagger.Schema do
     :additionalProperties,
     :discriminator,
     :example,
+    :examples,
```

Comment:
In swagger 2.0, only a single example is allowed (it was extended in swagger 3.0).
https://swagger.io/specification/v2/#schemaObject


Field Name | Type | Description
-- | -- | --
example | Any | A free-form property to include an example of an instance for this schema.